src/utils/generateuuid.js

```diff
@@ -8,8 +9,10 @@ const generateUUID = function generateUUID() {
     d += performance.now(); // use high-precision timer if available
   }
   const uuid = 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, (c) => {
+    // eslint-disable-next-line no-bitwise
     const r = (d + Math.random() * 16) % 16 | 0;
     d = Math.floor(d / 16);
+    // eslint-disable-next-line no-bitwise
     return (c === 'x' ? r : (r & 0x3 | 0x8)).toString(16);
```

Comment:
There is a better solution to this rather than no-bitwise.
(c === 'x' ? r : ((r && 0x3) || 0x8)).toString(16);

Reply:
This is correct according to [spec](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Bitwise_Operators)